scene/gui/shortcut.h

```diff
@@ -37,20 +37,20 @@
 class Shortcut : public Resource {
 	GDCLASS(Shortcut, Resource);
 
-	Ref<InputEvent> shortcut;
+	Ref<InputEvent> event;
 
 protected:
 	static void _bind_methods();
 
 public:
-	void set_shortcut(const Ref<InputEvent> &p_shortcut);
-	Ref<InputEvent> get_shortcut() const;
-	bool is_shortcut(const Ref<InputEvent> &p_event) const;
-	bool is_valid() const;
+	void set_event(const Ref<InputEvent> &p_shortcut);
+	Ref<InputEvent> get_event() const;
+	bool matches_event(const Ref<InputEvent> &p_event) const;
+	bool has_event() const;
 
 	String get_as_text() const;
 
-	Shortcut();
+	Shortcut() {}
```

Comment:
Shouldn't this be removed?

Reply:
I guess it could. I'm not 100% clear on all the nuances between not defining a constructor, defining an empty one, or a ` = default` one. But I guess in this context the nuances don't matter.